client/client.go

```diff
@@ -0,0 +1,105 @@
+package client
+
+import (
+	"errors"
+	"log"
+
+	pb "github.com/c-fs/cfs/proto"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+type Client struct {
+	conn *grpc.ClientConn
+}
+
+func New(address string) *Client {
```

Comment:
Return an error please. We normal do not fatal or panic inside a pkg when possible. 
Especially in this case, the user of pkg cannot know if the address is available/reachable before calling New(). Fatal is not a good option here.